larrecodnn/NuGraph/Tools/CMakeLists.txt

```
@@ -1,5 +1,7 @@
 cet_enable_asserts()
 
+cet_make_library(LIBRARY_NAME NuGraphBaseTools INTERFACE SOURCE LoaderToolBase.h DecoderToolBase.h)
+
```

You need to add the transitive dependencies to the `cet_make_library()` call with (e.g.)
```suggestion
cet_make_library(LIBRARY_NAME NuGraphBaseTools
  INTERFACE SOURCE LoaderToolBase.h DecoderToolBase.h
  LIBRARIES INTERFACE art::Framework_Core art_plugin_support::toolMaker art_plugin_types::tool
  art::Framework_Principal fhiclcpp::fhiclcpp
  lardataobj::RecoBase
 )
```

You might also want to survey the dependencies of the other libraries in that directory to see if any of the above may be removed as not used by the relevant headers and/or translation units.